roofit/histfactory/src/ParamHistFunc.cxx

```diff
@@ -10,13 +10,30 @@
 
 ////////////////////////////////////////////////////////////////////////////////
 
-/*
-BEGIN_HTML
-<p>
-</p>
-END_HTML
-*/
-//
+/** \class ParamHistFunc
+ * \ingroup HistFactory
+ *   A class which maps the current values of a RooRealVar
+ *  (or a set of RooRealVars) to one of a number of RooRealVars:
+
+ *  ParamHistFunc: {val1, val2, ...} -> {gamma (RooRealVar)}
```

Comment:
maybe this looks better enclosed in reverse quotes?

Reply:
Yeah ...
A comment I didn't touch at all. Just pulled it to the right place ...

Going the extra mile for you now ...